deploy/docker/README.md

```diff
@@ -0,0 +1,26 @@
+### Run localkube in a docker container (experimental)
+
+**Warning:** This is very experimental code at the moment.
```

Comment:
Can you add a note that this will only work on linux as well? The volume mounts won't work from OSX.